src/Stratis.Bitcoin.Tests/Utilities/AsyncManualResetEventTest.cs

```diff
@@ -33,7 +33,6 @@ public class AsyncManualResetEventTest
             stopwatch.Stop();
 
             Assert.True(stopwatch.ElapsedMilliseconds >= 500);
-            Assert.True(mreAwaitingTask.IsCompletedSuccessfully);
```

Comment:
strange, why does this make the test fail sometimes?

Reply:
I have no idea 🙂 
Before the last fix it was a check for State == RunToCompletion and it also used to fail. This is probably something environmental because I never had this test failing on win10 machine.

Reply:
Hmm. Actually that might be a problem of the stopwatch because it's still failing

Reply:
Yeah, the problem was in stopwatch- it sometimes shows 499ms passed instead of 500ms.